wicket/src/state/rack.rs

```diff
@@ -128,9 +125,8 @@ impl RackState {
             ComponentId::Sled(16) => ComponentId::Switch(1),
             ComponentId::Sled(0) => ComponentId::Sled(31),
             ComponentId::Sled(i) => ComponentId::Sled(i - 1),
-            // Skip over Psc(1) because it is always empty in currently shipping
-            // racks.
-            ComponentId::Switch(1) => ComponentId::Psc(0),
+            ComponentId::Switch(1) => ComponentId::Psc(1),
+            ComponentId::Psc(1) => ComponentId::Psc(0),
```

Comment:
Do we have a test to ensure that `next` and `prev` cancel each other out? Probably valuable to have that.